lib/http_adapter/angular.js

```diff
@@ -8,8 +8,13 @@
     function AngularAdapter($http, $window) {
         this._$http = $http;
         this._$window = $window;
+        this._timeout = 10000;
     }
 
+    AngularAdapter.prototype.setTimeout = function(timeout) {
```

Comment:
It seems odd to use `setTimeout` as a method name when it's also a built-in JS function... I can't think of a better one though.